packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -53,10 +55,29 @@ export abstract class InversifyHttpAdapter<
     return httpAdapterOptions.logger;
   }
 
+  #buildCorsOptions(
+    httpAdapterOptions: HttpAdapterOptions | undefined,
+  ): CorsOptions {
+    if (
+      httpAdapterOptions?.cors === undefined ||
+      typeof httpAdapterOptions.cors === 'boolean'
+    ) {
+      return {
+        methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
```

Comment:
Should we add `OPTIONS`? I think some browsers do options call before doing some cors calls, I think allowing OPTIONS would be nice